tool/pom.xml

```diff
@@ -1,133 +1,78 @@
-        <url>https://github.com/antlr/antlr4/issues</url>
-    </issueManagement>
-
-    <scm>
-        <url>https://github.com/antlr/antlr4/tree/master</url>
-        <connection>scm:git:git://github.com/antlr/antlr4.git</connection>
-        <developerConnection>scm:git:git@github.com:antlr/antlr4.git</developerConnection>
-        <tag>HEAD</tag>
-    </scm>
-
-    <distributionManagement>
-        <snapshotRepository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-        </snapshotRepository>
-        <repository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-        </repository>
-    </distributionManagement>
-
-
-    <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.11</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.seleniumhq.selenium</groupId>
-            <artifactId>selenium-java</artifactId>
-            <version>2.44.0</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>jetty-server</artifactId>
-            <version>8.1.16.v20140903</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.antlr</groupId>
-            <artifactId>antlr4-runtime</artifactId>
-            <version>4.5</version>
-        </dependency>
-        <dependency>
-            <groupId>org.antlr</groupId>
-            <artifactId>antlr-runtime</artifactId>
-            <version>3.5.2</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.antlr</groupId>
-            <artifactId>ST4</artifactId>
-            <version>4.0.8</version>
-            <scope>compile</scope>
-        </dependency>
-    </dependencies>
-
-
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.antlr</groupId>
+    <artifactId>antlr4-master</artifactId>
+    <version>4.5.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>antlr4</artifactId>
+  <name>ANTLR 4 Tool</name>
+  <description>The ANTLR 4 grammar compiler.</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>2.44.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
```

Comment:
This dependency is required in order to compile and run tests when creating a project from the pom.
It should not be removed.

> Le 20 juin 2015 à 02:26, Sam Harwell <notifications@github.com> a écrit :
> 
> In tool/pom.xml <https://github.com/antlr/antlr4/pull/920#discussion_r32855769>:
> 
> > +
> > +  <dependencies>
> > +    <dependency>
> > +      <groupId>junit</groupId>
> > +      <artifactId>junit</artifactId>
> > +      <version>4.11</version>
> > +      <scope>test</scope>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.seleniumhq.selenium</groupId>
> > +      <artifactId>selenium-java</artifactId>
> > +      <version>2.44.0</version>
> > +      <scope>test</scope>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.eclipse.jetty</groupId>
>  This dependency is not used by the build. It should be removed in order to avoid a large unnecessary download.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/920/files#r32855769>.
>